app/models/user.rb

```diff
@@ -307,6 +307,20 @@ def cached_followed_tag_names
         id: Follow.where(
           follower_id: id,
           followable_type: "ActsAsTaggableOn::Tag",
+          points: 1..,
+        ).select(:followable_id),
+      ).pluck(:name)
+    end
+  end
+
+  def cached_antifollowed_tag_names
```

Comment:
Yes, there's repetition between these two methods. No, I don't think that's a problem.